From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 5 Jun 2026 17:15:16 +0800
Subject: locale-en_DK

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Improve en_DK and add en_DK.ISO-8859-15 to SUPPORTED
# DP:   This locale is only useful to provide ISO8601 date formats.
# DP: Related bugs: #323159
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Jakob Bohm <jbj@image.dk>
# DP: Upstream status: not submitted
# DP: Date: 2006-01-16

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Improve en_DK and add en_DK.ISO-8859-15 to SUPPORTED
# DP:   This locale is only useful to provide ISO8601 date formats.
# DP: Related bugs: #323159
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Jakob Bohm <jbj@image.dk>
# DP: Upstream status: not submitted
# DP: Date: 2006-01-16
---
 localedata/SUPPORTED     |  1 +
 localedata/locales/en_DK | 22 +++-------------------
 2 files changed, 4 insertions(+), 19 deletions(-)

diff --git a/localedata/SUPPORTED b/localedata/SUPPORTED
index e9c1a97..6781f89 100644
--- a/localedata/SUPPORTED
+++ b/localedata/SUPPORTED
@@ -137,6 +137,7 @@ en_BW/ISO-8859-1 \
 en_CA.UTF-8/UTF-8 \
 en_CA/ISO-8859-1 \
 en_DK.UTF-8/UTF-8 \
+en_DK.ISO-8859-15/ISO-8859-15 \
 en_DK/ISO-8859-1 \
 en_GB.UTF-8/UTF-8 \
 en_GB/ISO-8859-1 \
diff --git a/localedata/locales/en_DK b/localedata/locales/en_DK
index 1632bbd..e88987f 100644
--- a/localedata/locales/en_DK
+++ b/localedata/locales/en_DK
@@ -63,27 +63,11 @@ copy "en_GB"
 END LC_CTYPE
 
 LC_MONETARY
-int_curr_symbol      "DKK "
-currency_symbol      "kr."
-mon_decimal_point    ","
-mon_thousands_sep    "."
-mon_grouping         3
-positive_sign        ""
-negative_sign        "-"
-int_frac_digits      2
-frac_digits          2
-p_cs_precedes        1
-p_sep_by_space       0
-n_cs_precedes        1
-n_sep_by_space       0
-p_sign_posn          1
-n_sign_posn          1
+copy "da_DK"
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        ","
-thousands_sep        "."
-grouping             3
+copy "da_DK"
 END LC_NUMERIC
 
 LC_TIME
@@ -153,7 +137,7 @@ copy "da_DK"
 END LC_MEASUREMENT
 
 LC_NAME
-name_fmt    "%d%t%g%t%m%t%f"
+copy "da_DK"
 END LC_NAME
 
 LC_ADDRESS
